src/chunkserver/op_request.h

```diff
@@ -150,7 +151,9 @@ class ChunkOpRequest : public std::enable_shared_from_this<ChunkOpRequest> {
      */
     static std::shared_ptr<ChunkOpRequest> Decode(butil::IOBuf log,
                                                   ChunkRequest *request,
-                                                  butil::IOBuf *data);
+                                                  butil::IOBuf *data,
+                                                  uint64_t index,
+                                                  PeerId GetLeaderId);
```

Comment:
`GetLeaderId` should be `leaderId` ? Also, `index` and `GetLeaderId` can be default parameters, so you don't have to modify all of this function call.

Reply:
In the CopysetNode::on_apply,follower must get index and leader peer.

Reply:
> In the CopysetNode::on_apply,follower must get index and leader peer.

In my opinion, the reason to let follower get `leader peer` is to avoid leader change cause send the follower's scan map information to a wrong leader, is my understanding correct? 
But what's the usage of the `index`?